kotest-framework/kotest-framework-standalone/build.gradle.kts

```diff
@@ -29,7 +23,11 @@ tasks {
       }
    }
    getByName("jvmJar") {
-      finalizedBy(getByName("shadowJar"))
+      finalizedBy(shadowJar)
+   }
+
+   startScripts {
+      dependsOn(shadowJar)
```

Comment:
The Gradle Application plugin creates a `.sh` and `.bat` scripts for the bundled application. `startScripts` is the task that generates those scripts.

The build config here is a bit hacky - instead of explicitly replacing the default JAR with the shadowed JAR, it changes the names of the JARs. However, Gradle doesn't realise this. `startScripts` tries to use the JAR produced by the default `jar` task, but because the name has changed, Gradle complains.